ios/uPortMobile/Classes/EthereumSigner.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018 ConsenSys AG
+//
+// This file is part of uPort Mobile App.
+//
+// uPort Mobile App is free software: you can redistribute it and/or modify
```

Comment:
Is it correct to call this `uPort Mobile App`, since it's not an app but SDK instead, and I'd expect `iOS` in the project/module name.